core/src/main/scala/org/apache/spark/deploy/worker/Worker.scala

```diff
@@ -362,11 +362,13 @@ private[spark] class Worker(
             self,
             workerId,
             host,
+            webUiPort,
```

Comment:
Argh, I shouldn't have let this get committed without making sure this comment had been addressed.  This is still broken; I'm fixing it as part of my own followup PR, which adds more tests.